cmd/plugin.go

```diff
@@ -108,6 +109,37 @@ var (
 			return mgr.Start(pluginName, exeArgs)
 		},
 	}
+
+	listCommand = &cobra.Command{
+		Use:     "list",
+		Short:   "List installed plugin",
+		Example: "vats plugin list",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			log.Info().Str("module", "plugin").Msgf("List plugins")
+
+			mgr := plugin.NewManager(pluginDir)
+			plugins, err := mgr.List()
+			if err != nil {
+				log.Error().Str("module", "plugin").Err(err)
+				return err
+			}
+
+			w := table.NewWriter()
+			w.SetOutputMirror(os.Stdout)
+			w.AppendHeader(table.Row{"Name", "Install Data", "Repository", "Version"})
+
+			for _, plugin := range plugins {
+				dateStr := plugin.InstalledAt.Format("2006-01-02 15:04:05")
```

Comment:
2006-01-02 15:04:05 ??

Reply:
@skonhwang This is unique style of Golang's date formatting.

Ref. https://pkg.go.dev/time#pkg-constants

Reply:
@rootwarp , Thanks to you I am more smarter about gland's   format of date